src/app/components/error-card/error-card.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
remove all spec.ts if you are not using them

Reply:
Got it